docs/reference/filters.md

```diff
@@ -1882,6 +1884,26 @@ Path("/cheap")     -> clusterLeakyBucketRatelimit("user-${request.cookie.Authori
 Path("/expensive") -> clusterLeakyBucketRatelimit("user-${request.cookie.Authorization}", 1, "1s", 5, 2) -> ...
 ```
 
+### ratelimitFailClosed
```

Comment:
Initially I was thinking about having a generic `ratelimitControl(config-key, config-value)` filter that we could re-use for other configs.
I guess a filter per specific config is also fine, moreover since in all ratelimit filters we fail open by default we can simplify and make this filter without arguments - users would add `ratelimitFailClosed()` or rely on the default.

Reply:
dropped all args